packages/ui/src/components/molecules/attachment/attachmentAdmin/attachmentAdmin.tsx

```
@@ -0,0 +1,76 @@
+            exit={{ opacity: 0, scale: 1 }}
+            initial={{ opacity: 0, scale: 0.95 }}
+            animate={{ opacity: 1, scale: 1 }}
+          >
+            <Card className="bg-[#26262666] p-4 border-0 flex gap-4">
+              <ScrollArea>
+                {files.map((file) => (
+                  <motion.div
+                    exit={{ opacity: 0, scale: 1 }}
+                    initial={{ opacity: 0, scale: 0.7 }}
+                    animate={{ opacity: 1, scale: 1 }}
+                  >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)